### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by nick-botticelli** [tihmstar/jbinit#6]
Will you be licensing this source code (such as for use in other projects)? 🙂

**Comment by tihmstar** [comment]
Yes, license was added here https://github.com/tihmstar/jbinit/commit/9e518055421626a91bde8a96c6876d25d3e183a2
